src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Entities/CacheableEntity.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace OrchardCore.Entities;
+
+public class CacheableEntity : ICacheableEntity
+{
+    private static readonly JObject _defaultProperties = new();
```

Comment:
again, this should not be used.

Reply:
See my comment [here](https://github.com/OrchardCMS/OrchardCore/pull/14337#discussion_r1327979180)